File: 0075-sort-colors/0075-sort-colors.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        zero_count = nums.count(0)
        one_count = nums.count(1)
        two_count = nums.count(2)
        idx = 0

        for i in range(zero_count):
            nums[idx] = 0
            idx += 1

        for i in range(one_count):
            nums[idx] = 1
            idx += 1

        for i in range(two_count):
            nums[idx] = 2
            idx += 1

File: 0075-sort-colors/0075-sort-colors.ts
/**
 Do not return anything, modify nums in-place instead.
 */
function sortColors(nums: number[]): void {
    let zeroCount: number = 0
    let oneCount: number = 0
    let twoCount: number = 0
    let idx: number = 0

    nums.forEach(function(num: number){
        if(num == 0){
            zeroCount++;
        } else if(num == 1){
            oneCount++;
        } else {
            twoCount++;
        }
    })

    for(let i = 0; i < zeroCount; i++){
        nums[idx++] = 0
    }

    for(let i = 0; i < oneCount; i++){
        nums[idx++] = 1
    }

    for(let i = 0; i < twoCount; i++){
        nums[idx++] = 2
    }
};